.env

```diff
@@ -0,0 +1,10 @@
+## The FQDN name this server will be reachable on
```

Comment:
Usually the `.env` file shouldn't be committed since it will change for every install (and then will make pulling updates via git more difficult).

It's a good idea to have a template in the repo though (e.g. `.env.template` or so)